MVVM/ViewModel/SettingsManager.cs

```diff
@@ -1,10 +1,17 @@
 using Godot;
 using RoverControlApp.Core;
+using RoverControlApp.Core;
+using RoverControlApp.MVVM.Model;
+using System.Threading.Tasks;
+using System;
 
 namespace RoverControlApp.MVVM.ViewModel;
 
 public partial class SettingsManager : Panel
 {
+	public event Func<Task>? OnSettingsApplied;
```

Comment:
```suggestion

```

Reply:
Adressed in [3775e21](https://github.com/wisniax/RoverControlApp/pull/16/commits/3775e2150e5da267c03c2f1e671f4844844b50f8)